sequana/bamtools.py

```diff
@@ -1502,6 +1526,45 @@ def hist_soft_clipping(self):
         pylab.xlabel("Soft clip length", fontsize=16)
         pylab.ylabel("#", fontsize=16)
 
+    def get_query_start(self, read):
```

Comment:
query_alignment_start and query_alignment_end from pysam are not doing the trick ?